stats/views.py

```diff
@@ -670,3 +695,23 @@ def get(self, request, *args, **kwargs):
         response = HttpResponse(contents, content_type='application/zip')
         response['Content-Disposition'] = 'attachment; filename={}'.format(zip_filename)
         return response
+
+
+class CaptchaTestView(generic.FormView):
+    form_class = CaptchaForm
+    template_name = 'stats/captcha.html'
+
+    def get_success_url(self):
+        return reverse('stats:scenarios')
+
+    def form_valid(self, form):
+        start_access_ts = time.time()
+        self.request.session['captcha-ts'] = start_access_ts
```

Comment:
Maybe add a comment that this is unused currently

Reply:
I forgot to remove this along with the other codes. Good that you spotted this.